src/main/java/com/daesy/songr/Album.java

```diff
@@ -8,6 +8,7 @@
     @Id
     @GeneratedValue (strategy = GenerationType.IDENTITY)
```

Comment:
You imports on this page should be:

`import javax.persistence.Entity;`
`import javax.persistence.GeneratedValue;`
`import javax.persistence.GenerationType;`
import javax.persistence.Id;